pinot-server/src/main/java/org/apache/pinot/server/api/resources/SegmentMetadataFetcher.java

```diff
@@ -55,6 +55,9 @@ private SegmentMetadataFetcher() {
   private static final String NULL_VALUE_VECTOR_READER = "null-value-vector-reader";
   private static final String RANGE_INDEX = "range-index";
   private static final String JSON_INDEX = "json-index";
+  private static final String H3_INDEX = "h3-index";
```

Comment:
can we add a test in `TableResourceTest.testSegmentMetadata` to indicate the behavior?

Reply:
Done.